src/coordinator/mod.rs

```diff
@@ -320,7 +320,11 @@ impl Coordinator {
                     self.process_inverter_packet(packet, &mut inputs_store)
                         .await?;
                 }
-                // this loop holds no state so doesn't care about inverter reconnects
+                Connected(serial) => match self.inverter_connected(serial).await {
```

Comment:
FIXME: neaten this up. It doesn't have to be a match, I'm sure.

My first attempt at this duplicated the `coordinator::commands::ReadHold` code inside `inverter.rs`. Adding the `Connected` state means we can drive it from the coordinator instead, which makes a lot more sense to me.

"Oh, the inverter is connected, I can coordinate all the tasks that happen on connection now"